Scarb.toml

```diff
@@ -1,11 +1,18 @@
-[package]
+[workspace]
+members = [
+    "src/data_structures",
+    "src/encoding",
+    "src/linalg",
+    "src/math",
+    "src/numeric",
+    "src/searching",
+    "src/sorting",
+    "src/storage"
+]
 name = "alexandria"
 version = "0.1.0"
-description = "Starknet utilities"
+description = "Community maintained Cairo and Starknet libraries"
 homepage = "https://github.com/keep-starknet-strange/alexandria/"
 
-[dependencies]
-starknet = ">=2.1.0"
-
```

Comment:
You can leave it here, and then use `starknet.workspace = true` in particular packages: https://docs.swmansion.com/scarb/docs/reference/workspaces.html#dependencies

Reply:
Getting this error:

```
error: failed to parse manifest at `/Users/m/projects/my-cairo-forks/alexandria/Scarb.toml`

Caused by:
    this virtual manifest specifies a [dependencies] section, which is not allowed
```

Reply:
Hi @milancermak!

This needs to be called `[workspace.dependencies]`.
https://docs.swmansion.com/scarb/docs/reference/workspaces.html#dependencies

Reply:
I'm dumb. Thanks.

Reply:
This is just confusing when you do not define the root package, but we mimic this distinction from Cargo manifest format :D

Reply:
I'm making a PR which add a hint how to fix this error FYI :D